isomerpages/isomercms-backend
# fix(quickie): reduceAmpBuildTimesCreateTests

[These tests will fail because dev tests are failing, interim solution in upstream pr #973]
## Problem

#965 has failing test cases. Had to convert js -> ts because I faced weird webpack issues that magically disappeared when shifting to TS. 

This does not add in new tests, but simply achieves parity with passing tests for #965. 

## Solution

Mostly changing APIs. 


## Tests
(same as #965) 
Manual tests in staging
Log into github: should be able to add a page to kishore-test-dev-gh and notice changes in staging and staging lite branch
log into email, should be able to add a page to kishore-test and notice changes in staging and staging lite.
All other repos, adding a page should have expected behaviour (adding an image should not throw errors + have a corresponding commit in github)
